PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
So Pillow targets almost every Python distribution? Not just the currently supported versions?